docs/site/Fields-filter.md

```diff
@@ -21,9 +21,16 @@ Where:
 - `<true|false>` signifies either `true` or `false` Boolean literal. Use `true`
   to include the property or `false` to exclude it from results.
 
+also can be used an array of strings with the properties
+
+<pre>
+{ fields: [<i>propertyName</i>, <i>propertyName</i>, ... ] }
+</pre>
+
 By default, queries return all model properties in results. However, if you
-specify at least one fields filter with a value of `true`, then by default the
-query will include **only** those you specifically include with filters.
+specify at least one fields filter with a value of `true` or put it in the
+array, then by default the query will include **only** those you specifically
+include with filters.
 
 ### REST API
```

Comment:
nitpick: @mdbetancourt could you also update the shortcut for the REST API section? thanks.